align/pnr/build_pnr_model.py

```diff
@@ -21,6 +21,7 @@ def _ReadVerilogJson( DB, j, add_placement_info=False):
 
         temp_node = PnR.hierNode()
         temp_node.name = module['name'] if 'name' in module else module['abstract_name']
+        temp_node.concrete_name = module['concrete_name'] if 'concrete_name' in module else module['name']
```

Comment:
hiernode needs concrete name, so that the placer can match placement json file to each hiernode

Reply:
@stevenmburns It seems the PnRDB is build in this function, not in C++ code. Is it possible to parse the concrete name in this function?